packages/iov-tendermint-rpc/src/requests.ts

```diff
@@ -102,6 +114,15 @@ export interface StatusRequest {
   readonly method: Method.STATUS;
 }
 
+export interface SubscribeRequest {
```

Comment:
See comment above in `adapter.ts`. I would like to merge these two interfaces into one, as they are not used in isolation, and this maintains consistency